.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on: [push]
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+    - name: Rubocop
+      run: |
+        gem install rubocop rubocop-performance --no-document
+        rubocop --require rubocop-performance \
+                --format progress \
+                --out ~/test-results/linting/rubocop.xml
```

Comment:
We can stop creating the output as a file. We are not using that.

So, remove the `--out ...` option.